models/edr/dbt_artifacts/alerts/alerts_dbt_models.sql

```diff
@@ -0,0 +1,31 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False
+  )
+}}
+
+with model_run_results as (
+    select * from {{ ref('model_run_results') }}
+),
+
+model_alerts as (
+    select model_execution_id as alert_id,
+           unique_id,
+           generated_at as detected_at,
+           database_name,
+           materialization,
+           path,
+           schema_name,
+           message,
+           owner as owners,
+           tags,
+           alias,
+           status,
+           full_refresh
+    from model_run_results
+    where lower(status) != 'success'
+)
+
+select * from model_alerts
+{{ dbt_utils.group_by(13) }}
```

Comment:
Do we need that group by?
It just returns the same rows (you group by everything you return) so isn't it just redundant calculation that can affect performance?

Reply:
I remember adding it because there were duplicates somehow, but now I removed it and there are none and it works perfectly without it. I removed it from the alerts of the tests now as well that `group_by(21)` which is pretty expensive.